### PR TITLE
antool: reformat 'for' loops

### PR DESCRIPTION
**Pull request by ldorau**
This change is [Reviewable](https://reviewable.io/reviews/pmem/pmemfile/269)

### CODE REVIEW

**Comment by ldorau** [comment]
@llugin please review

**Comment by krzycz** [comment]
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/pmemfile/269)*

**Comment by ldorau** [comment]
Review status: 1 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[src/tools/antool/listsyscalls.py, line 33 at r1](https://reviewable.io:443/reviews/pmem/pmemfile/269#-KptDsuxP--NPpqL5eXn-r1-33:-KptZ7w29PsyBtpRxA-w:b-90cdj) ([raw file](https://github.com/pmem/pmemfile/blob/9cb78a0a433b94698e7cba9d958f1864dd055f9d/src/tools/antool/listsyscalls.py#L33)):*
<details><summary><i>Previously, llugin wrote…</i></summary><blockquote>

It's better not to import whole modules but rather only parts that are actually used. I would change it to 'from os import path'.
</blockquote></details>

Removed at all

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/pmemfile/269)*

File: src/tools/antool/listsyscalls.py [hunk 1]
```diff
@@ -204,31 +203,30 @@ def fd_table_assign(table, fd, val):
...
     # look_for_matching_record -- look for matching record in a list of incomplete syscalls
     ####################################################################################################################
     def look_for_matching_record(self, info_all, pid_tid, sc_id, name, retval):
-        for n in range(len(self)):
-            syscall = self[n]
+        for syscall in self:
             check = syscall.check_read_data(info_all, pid_tid, sc_id, name, retval, DEBUG_OFF)
             if check == CHECK_OK:
-                del self[n]
+                self.__delitem__(self.index(syscall))
```
Comment (llugin) [inline src/tools/antool/listsyscalls.py]: I think self.remove(syscall) would be better

**Comment by ldorau** [comment]
Review status: 1 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/tools/antool/listsyscalls.py, line 221 at r2](https://reviewable.io:443/reviews/pmem/pmemfile/269#-KptDsuxP--NPpqL5eXn-r2-221:-KptknVPpUSiCWgfI2Q2:b-vc29ky) ([raw file](https://github.com/pmem/pmemfile/blob/620dd446f3cc7d747de340a06c6383fda52d88f3/src/tools/antool/listsyscalls.py#L221)):*
<details><summary><i>Previously, llugin wrote…</i></summary><blockquote>

I think self.remove(syscall) would be better
</blockquote></details>

That's what I was looking for... :-) Thanks! Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/pmemfile/269)*

**Comment by llugin** [comment]
:lgtm:

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/pmemfile/269#-:-Kptm8-PHIYJrmlT86jp:bnfp4nl)*

**Comment by ldorau** [comment]
@llugin please review the last change

**Comment by marcinslusarz** [comment]
:lgtm:

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r4.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/pmemfile/269#-:-Kpu3uoeE89atpqi1tgD:bnfp4nl)*

**Comment by llugin** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/pmemfile/269#-:-KpuBx-57vVi2ATN5aGw:bnfp4nl)*

**Pull request merged**